tools/monorepo-utils/src/code-freeze/utils/__tests__/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import { verifyDay } from '../index';
+
+describe( 'verifyDay', () => {
+	it( 'should return a string', () => {
+		expect( verifyDay() ).toBe( 'Today is a good day to code freeze!' );
```

Comment:
This PR is the result of a discussion in yesterday's team meeting. The idea was that we can ship the base for the tool that @psealock is using to migrate the code freeze action into. I was going to create very much the same package as this but since we're both needing the same thing this works out best.